src/FormBuilder/Routes/CreateFormRoute.php

```diff
@@ -30,18 +30,14 @@ public function __invoke()
                 exit();
             }
             if ('new' === $_GET['donationFormID']) {
-                $blocksJson = file_get_contents(
-                    GIVE_PLUGIN_DIR . 'src/FormBuilder/resources/js/form-builder/src/blocks.json'
-                );
-
                 $form = new DonationForm([
                     'title' => __('GiveWP Donation Form', 'give'),
                     'status' => DonationFormStatus::DRAFT(),
                     'settings' => FormSettings::fromArray([
                         'enableDonationGoal' => true,
                         'goalAmount' => 1000,
                     ]),
-                    'blocks' => BlockCollection::fromJson($blocksJson)
+                    'blocks' => (new GenerateDefaultDonationFormBlockCollection())(),
```

Comment:
We need to add an unreleased tag about these changes.

Reply:
resolved a2ce5081